python/hsfs/training_dataset.py

```diff
@@ -321,6 +321,9 @@ def insert(
     ):
         """Insert additional feature data into the training dataset.
 
+        !!! warning "Deprecated"
+        `insert` method is deprecated.
```

Comment:
This will break the docs. the line after the `!!!` needs to be indented with four spaces.